tools/src/main/java/org/wildfly/swarm/tools/DependencyManager.java

```diff
@@ -233,7 +237,17 @@ private void analyzeModuleDependencies() {
 
     private void analyzeModuleDependencies(ModuleAnalyzer analyzer) {
         List<ArtifactSpec> thorntailDependencies = analyzer.getDependencies();
-        this.moduleDependencies.addAll(thorntailDependencies);
+
+        // ModuleAnalyzer looks for dependencies in a predefined location and assumes that the folder conforms to the
+        // Maven repository layout. If it doesn't find the dependency in that location, then the ModuleAnalyzer will
+        // return an ArtifactSpec object that does not contain the file location. But when using the Gradle plugin, it
+        // is highly possible that the dependency was already retrieved and stored in the Gradle dependency cache.
+
```

Comment:
Please review this change after going through the implementation in [ModuleAnalyzer](https://github.com/thorntail/thorntail/blob/master/tools/src/main/java/org/wildfly/swarm/tools/ModuleAnalyzer.java#L86).